.github/workflows/coverage.yml

```diff
@@ -1,4 +1,4 @@
-name: Coverage CI
+name: 📈 • Coverage
```

Comment:
Just to double-check: The ``CI`` was intentionally deleted, right?

Reply:
Yes, I wanted to reserve `CI` for the actual macOS, windows, ubuntu and pyfiction CI's and use, e.g., 📈 • Coverage and 📝 • CodeQL for the others.

Reply:
We're often using ☂️ for coverage due to Codecov's logo. Do you think that would work here, too?

Reply:
Sure, sounds good!